index.html

```diff
@@ -61,5 +70,166 @@ <h1>Hey There.<br>I'm Daniel</h1>
+                            </ul>
+                    </div>
+                            <button class="card-btn">See Project</button>
+                </div>
+
+                <div class="card item6">
+                    <div class="text-block">
+                        <h3 class="new">Profesional Art<br>Printing Data</h3>
+                        <p class="card-description">A daily selection of privately 
+                            personalized reads; no accounts or 
+                            sign-ups required. has been the industry's standard</p>
+                            <ul class="lang">
+                                <li><a href="@" class="btn btn-car">html</a></li>
+                                <li><a href="@" class="btn btn-car">bootstrap</a></li>
+                                <li><a href="@" class="btn btn-car">Ruby</a></li>
+                            </ul>
+                    </div>
+                            <button class="card-btn">See Project</button>
+                </div>
+            </div>
+            </div>
+    </section>
+
+        <section class="about-section">
+            <div class="about">
+                <h1 class="spaced">About me</h1>
+                <p class="about-descript">Hello I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</p>
+                    <button class="abt-btn">Get My Resume</button>
+                    <img src="images/div.png" alt=""/>
+            </div>
+            <div class="about-2">
+
+                <div class="stacks">
+                    <img src="./images/polygon.png" alt=""/><span>Language</span>
+                    <ul>
+                        <li>Javascript</li>
+                        <li>Ruby</li>
+                        <li>Html</li>
+                        <li>Css</li>
+                    </ul>
+                </div>
+                <div class="frame">
+                    <img src="./images/square.png" alt=""/><span>Frameworks</span>
+                    <ul>
+                        <li>Booststrap</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSPec</li>
+                        <li>CapyBara</li>
+                        <li>Selenium</li>
+                    </ul>
+                </div>
+                <div class="skills">
+                    <img src="./images/circle.png" alt=""/><span>Skill</span>
+                    <ul>
+                        <li>Codekit</li>
+                        <li>Github</li>
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
+                    </ul>
+                </div>
+            </div>
+        </section>
 </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
I will the design comparisons at the end

| Figma design | Your design |
| --- | --- |
| <img width="284" alt="dfg" src="https://user-images.githubusercontent.com/4247740/143720164-27fb9a0f-f612-40eb-84dc-96cf3f1e45ec.PNG"> | <img width="284" alt="dfg2" src="https://user-images.githubusercontent.com/4247740/143720189-a561b30b-ed5a-4cc8-8f25-e873b7ba1deb.PNG"> |
| ✔️ | - I suggest that you compare the margins between elements and adjust them to meet the figma design. |
| <img width="283" alt="abc2" src="https://user-images.githubusercontent.com/4247740/143720353-6e6224e8-dc0f-49da-a436-fb6c7edc0202.PNG"> | <img width="284" alt="abc" src="https://user-images.githubusercontent.com/4247740/143720317-46dc625a-10b3-4e24-82d6-3a7d7e75a9c2.PNG"> |
| ✔️ | - As you can see, our figma design does not have a red border. I therefor request that we remove from the element so that it matches the design on the left. |